DOCS/man/options.rst

```diff
@@ -4115,6 +4115,33 @@ Demuxer
     all. The default is ``auto``, which behaves like ``recursive`` with
     ``--shuffle``, and like ``lazy`` otherwise.
 
+``--autocreate-playlist=<no|any|exts|same>``
+    When opening a local file, act as if the parent directory is opened and
+    create a playlist automatically.
+
+    :no:   Load a single file (default).
+    :any:  Create a playlist from the parent directory with any file type.
+    :exts: Create a playlist from the parent directory with files matching
+           extensions from any list ``autocreate-playlist-*-exts``.
+    :same: Create a playlist from the parent directory with files matching the
+           same category as the currently loaded file. One of the
+           ``autocreate-playlist-*-exts`` is selected based on the input file
+           and only files with matching extensions are added to the playlist.
+           If the input file itself is not matched to any extension list,
+           the playlist is not autogenerated.
+
+``--autocreate-playlist-video-exts=<ext-list>``
+    Video file extension list that is used to match files when using
+    ``--autocreate-playlist=<exts|same>``.
+
+``--autocreate-playlist-audio-exts=<ext-list>``
+    Audio file extension list that is used to match files when using
+    ``--autocreate-playlist=<exts|same>``.
+
+``--autocreate-playlist-image-exts=<ext-list>``
+    Image file extension list that is used to match files when using
+    ``--autocreate-playlist=<exts|same>``.
+
```

Comment:
`autocreate-playlist-*-exts` is string list options, need add docs for it.

Reply:
done